gradio/routes.py

```diff
@@ -292,7 +292,7 @@ def build_resource(path: str):
         async def favicon():
             blocks = app.get_blocks()
             if blocks.favicon_path is None:
-                return static_resource("img/logo.svg")
+                return static_resource("favicon.svg")
```

Comment:
Rather than adding a new favicon file - can we use pathlib here to construct the path correctly?

Reply:
Yes let's do that, you're right